node/src/PipeTransport.ts

```diff
@@ -310,7 +310,11 @@ export class PipeTransport
 
 		const consumer = new Consumer(
 			{
-				internal,
+				internal :
+					{
+						...this.internal,
```

Comment:
What is this.internal here and why do we need to pass it to the JS Consumer class? I mean, why do we have this.internal in Transport instead of just this.handerId and why does the JS Consumer needs to know about its parent's transportId? JS Consumer should be able to communicate with the Worker by just sending its Consumer handlerId, am I wrong? or is it for requests sent from the JS Consumer that must be handled by Transport in the Worker?

Reply:
It is exactly the same as before, just moved code from line 290 above to here because this is the only place it was needed now.